feat: created header, style: eslint refactoring

Roamdev opened the pull request.

Roamdev committed:
feat: created header, style: eslint refactoring

kn9ka commented on src/pages/Home/ui/Header.module.css:
```diff
@@ -0,0 +1,40 @@
+    transition: background-color 200ms ease; /* easeInQuart */
+}
+
+.link:hover {
```
мне не нравится такой вариант.
подсвети просто текст каким-нибудь градиентом.

kn9ka commented on src/pages/Home/ui/Header.module.css:
```diff
@@ -0,0 +1,40 @@
+.header {
+    display: flex;
+    position: fixed;
```
у тебя тут баг.
<img width="1102" alt="image" src="https://github.com/user-attachments/assets/49d5014e-6e42-493e-a0d2-5f08ffac30d9" />

Roamdev committed:
update header

исправил хеддер, добавил его в лайаут, добавил стили лайауту, всё отображается нормально

Co-Authored-By: Anton <13977991+kn9ka@users.noreply.github.com>
